thoth/package_extract/image.py

```diff
@@ -26,6 +26,13 @@
 from shlex import quote
 import hashlib
 from pathlib import Path
+from elftools.elf.elffile import ELFFile  # type: ignore
+from elftools.common.exceptions import ELFError  # type: ignore
+from packaging import version
+import tempfile
+import shutil
+import zipfile
+import tarfile
```

Comment:
Some of the imports added like`ELFError`, `tempfile` and `version` are probably unused.